clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1106,11 +1107,13 @@ public void assign(Collection<TopicPartition> partitions) {
             } else if (partitions.isEmpty()) {
                 this.unsubscribe();
             } else {
-                for (TopicPartition tp : partitions) {
+                partitions.forEach(tp -> {
```

Comment:
It would be nice to make related changes only :)

Reply:
OK. I have reverted un-related changes.